packages/restapi/src/lib/chat/helpers/inbox.ts

```diff
@@ -52,7 +52,11 @@ export const getInboxLists = async (
         toDID: ''
       }
     }
-    feeds.push({ ...list, msg: message, groupInformation: list.groupInformation });
+    Promise.all([message])
```

Comment:
This should be `await` Promise.all since we need to wait for the call

Reply:
Thanks for the input. I'll make the necessary changes. I thought of this way earlier but opted to use the .then() function. Could you please let me know what happens when we use the .then() function compared to the await keyword?

Reply:
The main reason is we want to wait for all messages to load but rather than doing it in a sequential form where 1 msg is queried after the previous one is fetched. We now do this in parallel and save some time.